test/unicode-spec.js

```diff
@@ -66,12 +66,12 @@ describe("unicode", function() {
         });
     });
 
-    it("should parse Latin-1 inside \\text{}", function() {
-        expect('\\text{ÀàÇçÉéÏïÖöÛû}').toParse();
+    it("should parse all lower case Greek letters", function() {
+        expect("αβγδεϵζηθϑικλμνξοπϖρϱςστυφϕχψω").toParse();
     });
 
-    it("should parse Latin-1 outside \\text{}", function() {
-        expect('ÀàÇçÉéÏïÖöÛû').toParse();
```

Comment:
Can we keep these tests but only check for `Çç`?  Eventually we'll want to test for `œ`, `Œ`, `æ`, `Æ`, and `ß` when they get added.

Reply:
This PR actually removes support for `Çç`.  I believe they were added prematurely when forcing support for Latin-1.  I just verified that none of the Latin-1 glyphs, `Çç` included, are in the KaTeX fonts.  (I believe this actually makes sense, and the purpose of this PR is to map them into the accent macros, like LaTeX would.)  Here's an example of incorrect looking `Çç` vs `Cc` in the current KaTeX:

![image](https://user-images.githubusercontent.com/2218736/33905005-31faa862-df4b-11e7-934a-c89809df7492.png)

Reply:
I opened https://github.com/KaTeX/katex-fonts/issues/2 to track the cedilla issue.